packages/squiggle-lang/src/rescript/Distributions/PointSetDist/PointSetDist.res

```diff
@@ -195,6 +195,18 @@ module T = Dist({
     | Discrete(m) => Discrete.T.variance(m)
     | Continuous(m) => Continuous.T.variance(m)
     }
+
+  let klDivergence = (t1: t, t2: t) =>
+    switch (t1, t2) {
+    | (Continuous(t1), Continuous(t2)) => Continuous.T.klDivergence(t1, t2)
+    | (Discrete(t1), Discrete(t2)) => Discrete.T.klDivergence(t1, t2)
+    | (Mixed(t1), Mixed(t2)) => Mixed.T.klDivergence(t1, t2)
```

Comment:
Are you sure this works? IF not, probably better to throw an error; we could patch later. (Strongly prefer less functionality to a occasionally broken math)

Reply:
I agree.
```rescript
switch (t1, t2) {
    | (Continuous(t1), Continuous(t2)) => Continuous.T.klDivergence(t1, t2)
    | _ => Error(NotYetImplemented)
    }
```